design-reference/docs/azure-simple.md

```diff
@@ -0,0 +1,206 @@
+| name | The name of the deployment.  This will be used across the resource created in this solution | string | n/a | yes |
+| resource\_group\_location | The deployment location of resource group container all the resources | string | n/a | yes |
+| subnet\_names | A list of public subnets inside the vNet. | list | `<list>` | no |
+| subnet\_prefixes | The address prefix to use for the subnet. | list | `<list>` | no |
+| subnet\_service\_endpoints | The list of service endpoints that will be given to each subnet | list | `<list>` | no ||
+
+
+### Module configuration
+
+<!---
+This content was generated automatically from `main.tf`.
+--->
+
+Each module in the `azure-simple` template contains its own configuration options. Click the links below to learn more.
+ - [`service-plan`](https://microsoft.github.io/orion/service-plan#Configuration)
+ - [`app-service`](https://microsoft.github.io/orion/app-service#Configuration)
+ - [`app-insights`](https://microsoft.github.io/orion/app-insights#Configuration)
+
+
+### Deployment steps
+
+Create a new Terraform template directory and add a `main.tf` file. Here's a sample that uses the `azure-simple` template.
+
+```
+Add sample here. Required.
+```
+
+Call the `terraform init`, `terraform plan`, and `terraform apply` commands within the template directory to initialize the Terraform deployment then write and apply the plan.
+
+```
+Add code block here. Required.
+```
+
+<!---
+This content will be generated by [TBD].
+--->
+
+The following resources will be created:
+
+ - Linux App Service Plan
+ - App Service Container + staging slot
+ - Application Insights Instance
+ - vnet + subnet for app gateway
+ - Application Gateway with an enabled Firewall
+ - Traffic Manager Profile
+ - Traffic Manager Endpoint
+ - Azure Monitoring Rules
+ - Public IP(for TM + AG)
+
+
+### Outputs
+
+<!---
+This content was generated automatically from `ouputs.tf` using `terraform-docs` (https://github.com/segmentio/terraform-docs).
+--->
+
+| Name | Description |
+|------|-------------|
+| app\_service\_default\_hostname |  |
+| app\_service\_name |  
+
+
+## Features
+
+<!---
+This content will be generated by [TBD].
+--->
+
+| Feature            | Description                        |
+|--------------------|------------------------------------|
+| Performance |  |
+| Scaling |  |
+| Cost | |
+| Security | |
+| Authentication | |
+| Authorization | |
+
+## In-depth guides
+
+### Design rationale
+
+Content about design rationale. Optional.
+
+### Testing
+
+<!---
+This content will be generated by [TBD].
+--->
+
+These are the basic steps for testing.
+  1. Step one.
+  2. Step two.
+  3. Step three.
+
+
+## References
+
+
+### Reference apps
+
+These sample applications can be deployed with `azure-simple`.
+
+<!---
+This content will be generated by [TBD].
+--->
+
+  - [Helium](https://github.com/microsoft/helium)
+
+
+### FAQ
+
+<!---
+This content will be generated by [TBD].
+--->
+ > **Q: Question one?**
```

Comment:
you may want to include a sample question like `Q. The template is taking long time to delete`